player/public/services/mainservice.js

```diff
@@ -4,6 +4,7 @@
 var RendererServices = function () {}
 window.org.ekstep.service = new RendererServices()
 RendererServices = undefined
+var isMobile = (/android|webos|iphone|ipad|ipod|blackberry|iemobile|opera mini/i.test(navigator.userAgent.toLowerCase()));
```

Comment:
Remove this code/logic. Use window.cordova if you still need to identify mobile

Reply:
If I use window. cordova here, then telemetry fails to initialize which leads to stopping the player.